docs/source-configuration.md

```diff
@@ -0,0 +1,59 @@
+Configuring sources
+===================
+
+Heapster can get data from multiple sources. These are specified on the command line
+via the `--source` flag. The flag takes an argument of the form `PREFIX:CONFIG[?OPTIONS]`.
+Options (optional!) are specified as URL query parameters, separated by `&` as normal.
+This allows each source to have custom configuration passed to it without needing to
+continually add new flags to Heapster as new sources are added. This also means
+heapster can capture metrics from multiple sources at once, potentially even multiple
+Kubernetes clusters.
+
+## Current sources
+### Kubernetes
+To use the kubernetes source add the following flag:
+
+```
+--source=kubernetes:<KUBERNETES_MASTER>[?<KUBERNETES_OPTIONS>]
+```
+
+If you're running Heapster in a Kubernetes pod you can use the following flag:
+
+```
+--source=kubernetes:https://${KUBERNETES_RO_SERVICE_HOST}:${KUBERNETES_RO_SERVICE_PORT}
```

Comment:
I think this can be just the dns name: `source=kubernetes:https://kubernetes-ro`. Does the SkyDNS setup work in OpenShift?